### ISSUE TITLE
Doubt on VGG19 features on the RAW images

### ISSUE THREAD

**Issue by sayakpaul** [MediaTek-NeuroPilot/mai21-learned-smartphone-isp#5]
[This line](https://github.com/MediaTek-NeuroPilot/mai21-learned-smartphone-isp/blob/master/train_model.py#L68) suggests it's using a pre-trained VGG19 on the RAW images to extract meaningful representations.

How come a model pre-trained on 3 channel images is being used here for this purpose?

Any clarification would be helpful.

**Comment by cmhungsteve** [comment]
VGG model is used for one of the loss functions `loss_content` in the baseline.
You can remove that loss if you don't want (line 65-72 in `train_model.py`)
I will update README for clarification.
Thank you for pointing out this confusion.

**Comment by sayakpaul** [comment]
Understood. But even if that is the case it's unclear as to how the VGG model is incorporated on channel-4 images.

**Comment by cmhungsteve** [comment]
`enhanced_vgg = vgg.net(vgg_dir, vgg.preprocess(enhanced * 255))`
the input of VGG is the output of PUNET (`enhanced`), so it's an RGB image, not RAW image.

**Comment by sayakpaul** [comment]
Okay. Got it. Thanks.
